.github/workflows/main.yml

```diff
@@ -70,6 +70,33 @@ jobs:
           file: build/*/*.dmg
           tag: ${{ github.ref }}
           file_glob: true
+          
+  appcast:
+    name: Set Appcast
+    needs: build
+    runs-on: macos-latest
+    env:
+      JOB_TYPE: BUILD
+      SIGNATURE_FOR_SIGNING: ${{ secrets.SIGNATURE_FOR_SIGNING }}
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Get Information & Sign
+        run: |
+           TAG_VER=${GITHUB_REF/refs\/tags\//}
+           DATE=$(date -R)
+           SIGNATURE=$(./bin/sign_update -s ${{ secrets.SIGNATURE_FOR_SIGNING }} build/Release/MaciASL-${TAG_VER}-RELEASE.dmg)
+           ./bin/generate_appcast -o appcast.xml -s ${{ secrets.SIGNATURE_FOR_SIGNING }} --download-url-prefix https://github.com/acidanthera/MaciASL/releases/download/${TAG_VER}/ build/Release || exit 1
```

Comment:
Is there no mistake here? I do not see `SIGNATURE` being used anywhere. If it is not needed and creates some file, it is best to have `./bin/sign_update -s ${{ secrets.SIGNATURE_FOR_SIGNING }} build/Release/MaciASL-${TAG_VER}-RELEASE.dmg`.

Reply:
Sorry, overlooked

Reply:
It probably needs some error checking too.

Reply:
`- name: Get Sparkle 1.26
        run: |
          curl -L -s "https://github.com/sparkle-project/Sparkle/releases/download/1.26.0/Sparkle-1.26.0.tar.xz" -o Sparkle.tar.xz
          tar -xf Sparkle.tar.xz`

This one too?

Reply:
Yes. `|| exit 1` feels good enough.